### PR TITLE
Sub-Release 0: Release/v0.6.0: Redesign of Battery Passport 

### PR DESCRIPTION
**Pull request by matbmoser**
# PR part of release v0.6.0, Sub Release v0.6.0.0

# Why we create this PR?
 
To merge the new redesign passport changes
 
# What we want to achieve with this PR?
 
Upgrade the actual design of the passport
 
# What is new?
 
## Added
- Refactor the UI according to the designs given by Accenture
- Implemented new User Interface (UI) design for battery passport page

### CODE REVIEW

File: src/assets/styles/components/passport/cards.scss
Comment (SebastianBezold) [inline src/assets/styles/components/passport/cards.scss:1]: Copyright and License Header missing. Consider to use your IDE capabilities to auto-add them on new files

File: src/assets/styles/components/passport/elementChart.scss
Comment (SebastianBezold) [inline src/assets/styles/components/passport/elementChart.scss:1]: Copyright + License header

File: src/assets/styles/components/passport/field.scss
Comment (SebastianBezold) [inline src/assets/styles/components/passport/field.scss:1]: Header again

File: src/assets/styles/components/passport/passportPage.scss
Comment (SebastianBezold) [inline src/assets/styles/components/passport/passportPage.scss:1]: Header

File: src/components/passport/BarChart.vue
Comment (SebastianBezold) [inline src/components/passport/BarChart.vue:1]: This license header does not seem to be complete.
At lease SPDX identifier missing. Eclipse contributors part missing

File: src/components/passport/Cards.vue
Comment (SebastianBezold) [inline src/components/passport/Cards.vue:1]: Again incomplete header

File: src/components/passport/ElementChart.vue
Comment (SebastianBezold) [inline src/components/passport/ElementChart.vue:1]: Incomplete header

**Review by SebastianBezold — CHANGES_REQUESTED** [review]
You do have a lot of copyright header findings again. Please use your IDE capabilities. It just takes too much time for a task, that could have been automated

**Comment by SebastianBezold** [comment]
@matbmoser can you also please ask someone to do a content review, since I cannot review the CSS and JavaScript changes.
Comments about the GitGuardian findings are also appreciated if you already analyzed them

**Comment by matbmoser** [comment]
> You do have a lot of copyright header findings again. Please use your IDE capabilities. It just takes too much time for a task, that could have been automated

We are aware of that... Unfortunately not all our developers have a automatic header license so we need to revise all manually and sometimes some files scape our control. They will be fixed.

**Comment by matbmoser** [comment]
https://github.com/eclipse-tractusx/digital-product-pass/pull/16#issuecomment-1520229972

GitGuardian is detecting false positives in the values.yaml

**Comment by matbmoser** [comment]
https://github.com/eclipse-tractusx/digital-product-pass/pull/16#issuecomment-1527244768

@SebastianBezold I will ask the question today in the DevSecOps meeting, I have it configured in Intelij however mostly we use VSCode and the structure provided limits us (because it does not includes the SPDX field and the header notice at the same time)

**Comment by matbmoser** [comment]
@SebastianBezold I just created a discussion about that in the tractus-x dev mailing list, so we can find the best practices in this case.

**Comment by matbmoser** [comment]
@SebastianBezold we solved the necessary fixes for the license headers in the last PR from the version v0.6.0 -> #24
Please review the PR and let us know your feedback.

**Comment by saudkhan116** [comment]
Hi @SebastianBezold, I have reviewed the JavaScript and CSS content and they are fine enough.

**Comment by matbmoser** [comment]
Sub release strategy (slit bigger releases) proposed by the System Team failed to work, because of lack of tracking and the lack of rebase possibilities. https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow


The new commit strategy will follow this diagram from gitflow:

image



Only one PR will be merged, the #24

**Comment by SebastianBezold** [comment]
Merging without copyright headers, but tracking in #24

**Pull request merged**